frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-install.tsx

```diff
@@ -0,0 +1,365 @@
+      </p>
+      <form className="co-m-pane__body-group" onSubmit={submit}>
+        <div className="form-group co-create-route__name">
+          <label className="co-required" htmlFor="select-node-help">
+            Select Nodes
+          </label>
+          <div className="control-label help-block" id="select-node-help">
+            A minimum of 3 nodes needs to be labeled with{' '}
+            <code>cluster.ocs.openshift.io/openshift-storage=&quot;&quot;</code> in order to create
+            the OCS Service.
+          </div>
+          <Alert
+            className="co-alert ceph-ocs-info__alert"
+            variant="info"
+            title="An AWS bucket will be created to provide the OCS Service."
+          />
+          <p className="co-legend co-required ceph-ocs-desc__legend">
+            Select at least 3 nodes you wish to use.
+          </p>
+          <ListPage kind={NodeModel.kind} showTitle={false} ListComponent={OCSNodeList} />
+          <p className="control-label help-block" id="nodes-selected">
+            {selectedNodesCnt} node(s) selected
+          </p>
+        </div>
+        <h4>OCS Service Capacity</h4>
+        <div className="form-group">
+          <label className="control-label" htmlFor="request-size-input">
+            Requested Capacity
+          </label>
+          <RequestSizeInput
+            name="requestSize"
+            required={false}
+            onChange={handleRequestSizeInputChange}
+            defaultRequestSizeUnit={requestSizeUnit}
+            defaultRequestSizeValue={requestSizeValue}
+            dropdownUnits={dropdownUnits}
+          />
+        </div>
+        <div className="form-group">
+          <Tooltip content="The Storage Class will be used to request storage from the underlying infrastructure to create the backing persistent volumes that will be used to provide the OpenShift Container Storage (OCS) service">
+            <OCSStorageClassDropdown
+              onChange={handleStorageClass}
+              id="storageclass-dropdown"
+              required={false}
+              name="storageClass"
+            />
+          </Tooltip>
+        </div>
+        <ButtonBar errorMessage={error} inProgress={inProgress}>
+          <ActionGroup className="pf-c-form">
+            <Button type="submit" variant="primary" isDisabled={createBtnDisabled}>
+              Create
+            </Button>
+            <Button type="button" variant="secondary" onClick={history.goBack}>
+              Cancel
+            </Button>
+          </ActionGroup>
+        </ButtonBar>
+      </form>
+    </div>
+  );
+});
+
+export const CreateOCSServiceYAML: React.FC<CreateOCSServiceYAMLProps> = (props) => {
+  const template = _.attempt(() => safeDump(props.sample));
+  if (_.isError(template)) {
+    // eslint-disable-next-line no-console
+    console.error('Error parsing example JSON from annotation. Falling back to default.');
+  }
+
+  return (
+    <CreateYAML
+      {...props as any}
```

Comment:
Why do we need `as any` here? This usually means a type declaration is wrong somewhere.